azure-iot-hub/azure/iot/hub/iothub_registry_manager.py

```diff
@@ -238,31 +231,11 @@ def delete_device(self, device_id, etag=None):
 
         self.protocol.service.delete_device(device_id, etag)
 
-    def get_service_statistics(self):
```

Comment:
>def get_service_statistics(self): [](start = 4, length = 33)

We don't need these?

Reply:
They have been moved to the bottom of the file

---
In reply to: [354966538](https://github.com/Azure/azure-iot-sdk-python/pull/397#discussion_r354966538) [](ancestors = 354966538)